SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -1315,11 +1315,8 @@ void USpatialActorChannel::ServerProcessOwnershipChange()
 		FWorkerComponentUpdate Update = NetOwningClientWorkerData->CreateNetOwningClientWorkerUpdate();
 		NetDriver->Connection->SendComponentUpdate(EntityId, &Update);
 
-		// Update the EntityACL component (if authoritative).
-		if (NetDriver->StaticComponentView->HasAuthority(EntityId, SpatialConstants::ENTITY_ACL_COMPONENT_ID))
-		{
-			Sender->UpdateClientAuthoritativeComponentAclEntries(EntityId, NewClientConnectionWorkerId);
-		}
+		// Notify the load balance enforcer of a potential short circuit if we are the ACL authoritative worker.
+		NetDriver->LoadBalanceEnforcer->ShortCircuitMaybeRefreshAcl(EntityId);
```

Comment:
I assume now we know we are ACL authoritive because the the SubView in the  lb enforcer will only contain entitys that we have auth over?

Reply:
If you check the short circuit function, it checks for the LB tag on the entity, indicating that we are auth-complete over that entity, and only does the ACL refresh if so. It might be worth adding being able to check that on the subview itself so we don't need to know the tag. Let me know if the flow is confusing, can always be improved.

Reply:
That make sense, I just wasn't sure how we enforce the "auth-complete" check. Using the Tag, coooool